src/coffee/helpers/sftp.coffee

```diff
@@ -18,19 +18,11 @@ class Sftp
   ###
   listFiles: (sftp, dirName) ->
     new Promise (resolve, reject) ->
-      sftp.opendir dirName, (err, handle) ->
-        if err
+      sftp.readdir dirName, (err, list) ->
+        if err?
```

Comment:
doesn't make a huge difference as I can't think of a reason why err should be `0`, changed it anyway: f04467a